subprojects/docs/src/docs/userguide/authoring-builds/plugins.adoc

```diff
@@ -169,14 +169,9 @@ Where `«plugin id»`, in case `#1` is a static Kotlin extension property, named
 `«plugin version»` is also a string.
 The `apply` statement with a `boolean` can be used to disable the default behavior of applying the plugin immediately (e.g. you want to apply it only in `subprojects`).
 
-The `plugins {}` block must also be a top level statement in the buildscript. It cannot be nested inside another construct (e.g. an if-statement or for-loop).
-
-[[sec:build_scripts_only]]
-===== Can only be used in build scripts
+See <<#sec:plugin_version_management,plugin version management>> if you want to use a variable to define a plugin version.
 
-The `plugins {}` block can currently only be used in a project's build script. It cannot be used in script plugins, the settings.gradle file or init scripts.
```

Comment:
Half of this is still true.  The `plugins {}` block cannot be used in script plugins or init scripts.  It can only be used in project build scripts and settings scripts.